applications/machine_learning/configuration/thingy53_nrf5340_cpuapp_ns/app_ZDebug.conf

```diff
@@ -0,0 +1,191 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+# Application configuration
+
+CONFIG_ML_APP_EI_DATA_FORWARDER=y
+CONFIG_ML_APP_EI_DATA_FORWARDER_BT_NUS=y
+CONFIG_ML_APP_EI_DATA_FORWARDER_SENSOR_EVENT_DESCR="accel_xyz"
+
+CONFIG_ML_APP_LED_STATE=y
+CONFIG_ML_APP_LED_STATE_VALUE_THRESH=700
```

Comment:
Please update the following values in app configurations:
```
CONFIG_BT_L2CAP_TX_MTU=498
CONFIG_BT_BUF_ACL_RX_SIZE=502
```

Reply:
Also make sure to update the following one in net core configuration
`CONFIG_BT_BUF_ACL_RX_SIZE=502`

Reply:
Needed to speed up DFU

Reply:
Shouldn't we make it in different PR? Its just about adding the configuration. We can create separate PR for speeding up the dfu.

Reply:
The DFU speed up was already added for cpuapp, so it should be added here too, Discussed offline.